client/network/src/service.rs

```diff
@@ -733,25 +733,116 @@ pub struct NetworkWorker<B: BlockT + 'static, H: ExHashT> {
 	/// Senders for events that happen on the network.
 	event_streams: Vec<mpsc::UnboundedSender<Event>>,
 	/// Prometheus network metrics.
-	metrics: Option<Metrics>
+	metrics: Option<Metrics>,
 }
 
 struct Metrics {
+	// This list is ordered alphabetically
+	average_download_per_sec: Gauge<U64>,
+	average_upload_per_sec: Gauge<U64>,
```

Comment:
It might be better to put these two together into a `GaugeVec` and use label values like we do here: https://github.com/paritytech/substrate/blob/c3ff85d13b8b660c2c4eb850209c06a9397bed3e/client/service/src/builder.rs#L1112-L1113

Same could be done for things like import queue submissions. @mxinden what do you think?

Reply:
Yes. Very good point. That makes later querying with PromQL a lot easier.